### ISSUE TITLE
Add search (live filter?) box for GraphQL schema panel

### ISSUE THREAD

**Issue by Qix-** [hoppscotch/hoppscotch#1188]
**Is your feature request related to a problem? Please describe.**

The schema fetching in Hoppscotch is very helpful and (mostly) well laid-out. However, it appears to be using a virtual list of some sort and thus isn't ctrl+f searchable, making large schemas incredibly hard to traverse.

This, paired with the fact it can't be resized (it's a tiny sliver for me) and that clicking types and whatnot doesn't seem to scroll to the appropriate positions, makes it for kind of a clunky experience - especially when the API in question doesn't provide formal docs and instead simply points to the in-schema documentation.

**Describe the solution you'd like**

Basically this.

![image](https://user-images.githubusercontent.com/885648/94298672-d005a780-ff66-11ea-8d5b-d32d1311eafb.png)

**Describe alternatives you've considered**

Nothing really, feels like this is a core feature that'd be abundantly useful.

Alternatively, a tree-like viewer instead of a long virtual list, where items can be expanded to include documentation notes and whanot (collapsed by default).

**Additional context**

Only appears to be missing for GraphQL. The component in the screenshot is just hacked in, copied from another tab.

**Comment by liyasthomas** [comment]
Thanks for reporting your pain points regarding GraphQL page. We're working on an overhaul on GraphQL page and hoping to bring the best experience. Will use this issue as a base to track it's progress.

**Comment by AndrewBastin** [comment]
I think this will be good for Hacktoberfest as I am preoccupied with working on the Teams backend.

The implementation is pretty trivial actually.

**Comment by LeoMartinDev** [comment]
I'll give it a try 👌

**Comment by LeoMartinDev** [comment]
I'm wondering, should it filters only on `fieldName` ? Maybe we could also search in the description, not sure about this

**Comment by Qix-** [comment]
Being able to fuzzy search for any of the content would be groundbreaking. Some of the schemas that get imported are _massive_ and it's not always clear what the field name will be.

**Comment by LeoMartinDev** [comment]
> Being able to fuzzy search for any of the content would be groundbreaking. Some of the schemas that get imported are _massive_ and it's not always clear what the field name will be.
> 
> Just my $0.02.

You mean, including mutations args, fields types, etc.. ? Got something workin, filtering on description & field/mutation/type name

**Comment by Qix-** [comment]
Yeah, basically any part of the spec that is returned would be searchable in theory - within reason, of course.

**Comment by LeoMartinDev** [comment]
> Yeah, basically any part of the spec that is returned would be searchable in theory - within reason, of course.

For now it only search in `name` and `description` but I made it easy to add more keys to search in
